### PR TITLE
New ncbi

### LINKED ISSUE #127: SraRunTable.txt downloads as CSV, not TSV
In the updated version of the SRA Run Selector Page, the downloaded SraRunTable.txt is actually now a comma-delimited file rather than a tab-delimited file as stated on the current version of the "Examining Data on the NCBI SRA Database" page.

 Students will need to specify that their spreadsheet program interpret it as comma-delimited, so I suggest the following language: "Using your choice of spreadsheet program, open the SraRunTable.txt file. You may need to tell the program that this is a comma-delimited file in order to have the data separated properly into columns."

### PR DESCRIPTION
**Pull request by dpshelio**
This reverts the revert of #129 and builds on top of #128, fixing the links that may have required the revert.

I've merged these changes [on my fork so I can demonstrate how it looks](https://dpshelio.github.io/organization-genomics/03-ncbi-sra/index.html) before this is merged.

This fixes #127

### CODE REVIEW

File: _episodes/03-ncbi-sra.md [hunk 1]
```diff
@@ -43,42 +43,49 @@ following 13 sub-projects:"**.
...
-2. Click on the Run Number of the first entry in the bottom table on the page (third table down). This will take you to a page that is a run browser. Take a few minutes to examine some of the descriptions on the page.
+6. Click on the “Metadata” button to download the data for this lesson. The filename is “SraRunTable.txt” and save it on your computer Desktop.
 
-3. Go back to the ['previous page'](https://trace.ncbi.nlm.nih.gov/Traces/study/?acc=SRP064605). At the top of the page and in the **Total** row you will see there are 312 runs, 109.43 Gb data, and 168.81 Gbases of data. Click the 'RunInfo Table' button and save the file locally.
+> ## Downloading the Lenski SRA data from the SRA Run Selector Table using the old Run Selector
+> 
+> Go to the [“old Run Selector” instructions](../old-ncbi/index.html) page and return here after downloading your `SraRunTable.txt` file.
+> 
+> We include the “old Run Selector” page because the webpages ***and*** downloaded file `SraRunTable.txt` are slightly different. 
+{: .callout}
```
Comment (dpshelio) [inline _episodes/03-ncbi-sra.md:79]: I've converted this into a `callout` so it's rendered in yellow to warn the users.

File: _episodes/03-ncbi-sra.md [hunk 2]
```diff
@@ -43,42 +43,49 @@ following 13 sub-projects:"**.
...
 
-We are not downloading any actual sequence data here! This is only a text file that fully describes the entire
-dataset.  
-
-You should now have a file called `SraRunTable.txt`
+**You should now have a file called `SraRunTable.txt`**
```
Comment (dpshelio) [inline _episodes/03-ncbi-sra.md:81]: I've changed this from a sub-section to text in bold

File: _extras/old-ncbi.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+![ncbi-run-browser.png](../fig/03_ncbi_run_browser.png)
+
+4. Use your browser’s “Back” button or arrow to go back to the ['previous page'](https://trace.ncbi.nlm.nih.gov/Traces/study/?acc=SRP064605). Above where it lists the "312 Runs found" is a line starting with **Total** and you will see there are 312 runs, 109.43 Gb data, and 168.81 Gbases of data. Click the 'RunInfo Table' button and save the file to your Desktop.
+![ncbi-old-runtable-button.png](../fig/03_ncbi_old_runtable_button.png)
+We are not downloading any actual sequence data here! This is only a text file that fully describes the entire
+dataset.  
+
+You should now have a **tab-delimited** file called `SraRunTable.txt`.
+
+**Return to lesson [Examining Data on the NCBI SRA Database](../03-ncbi-sra/index.html#you-should-now-have-a-file-called-sraruntabletxt) and continue.**
```
Comment (dpshelio) [inline _extras/old-ncbi.md:26]: I've changed the link so it works properly when rendered.

File: _episodes/03-ncbi-sra.md [hunk 4]
```diff
@@ -43,42 +43,49 @@ following 13 sub-projects:"**.
...
+![ncbi-new-metadata.png](../fig/03_ncbi_new_metadata.png)
 
-2. Click on the Run Number of the first entry in the bottom table on the page (third table down). This will take you to a page that is a run browser. Take a few minutes to examine some of the descriptions on the page.
+6. Click on the “Metadata” button to download the data for this lesson. The filename is “SraRunTable.txt” and save it on your computer Desktop.
 
-3. Go back to the ['previous page'](https://trace.ncbi.nlm.nih.gov/Traces/study/?acc=SRP064605). At the top of the page and in the **Total** row you will see there are 312 runs, 109.43 Gb data, and 168.81 Gbases of data. Click the 'RunInfo Table' button and save the file locally.
+> ## Downloading the Lenski SRA data from the SRA Run Selector Table using the old Run Selector
+> 
+> Go to the [“old Run Selector” instructions](../old-ncbi/index.html) page and return here after downloading your `SraRunTable.txt` file.
```
Comment (dpshelio) [inline _episodes/03-ncbi-sra.md:76]: I'd changed the link so it's not absolute to the organisation.

**Review by dpshelio — COMMENTED** [review]
These are the changes I've made from what @hoytpr had made in #128. Or see them independently on 03be65c619b66c03892c4e56ebf5be574c15fe50

**Comment by hoytpr** [comment]
Thanks for all your help @dpshelio and @ewallace! The lesson is much improved by your attention to details. The "anchor" from the old NCBI page to the new NCBI lesson page doesn't seem to be working, but otherwise everything looks great. The anchor works when I render the pages locally on my Win10 machine, but not on GitHub. Maybe we can still work on that.

**Pull request merged**